runner.py

```diff
@@ -0,0 +1,134 @@
+import runner_run_funcs
+
+SIM_TYPE_FED_LEARNING = "fed_learning"
```

Comment:
These constants don't have a global semantic meaning.  Let's scope these to be class attributes of `Runner`

Reply:
Yeah this makes sense once we get rid of the other classes in the module.

Reply:
I'm wondering now if it actually does make more sense for these to be loose in `runner.py` (or maybe even in a file of their own).

Currently there are multiple files that are dependent on these keys, `runner.py` itself and `runner_run_funcs.py`, and also now any code that's going to be constructing the json. It feels kind of weird to me for these to be defined as statics on `Runner` while in `runner_run_funcs` we don't care at all about the `Runner` class besides these static keys that live on it.

This is kind of nit picky, so IDK. I'd be interested to know your thoughts on this.

Reply:
It's also kind of weird since both `runner.py` and `runner_run_funcs.py` are deps of eachother.

Reply:
There is a dependency issue happening when importing `runner.py`.

`runner` imports `runner_run_funcs`, which needs the static key variables from Runner (class), but runner hasn't been fully imported yet so it doesn't see it as existing.

I can't see a nice way for these to be static without this issue. There might be (if there is, just post a comment here and I'll make the changes it), but for now I'm going to put the keys loose in their own file.

Reply:
Can we just cheat and just those two modules together for now?

Reply:
Haha sure. :)